Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+"discord.py" = "==1.6.0"
```

Comment:
From pipenv's docs:
> The use of ~= is preferred over the == identifier as the latter prevents pipenv from updating the packages

We should update these accordingly

Reply:
ok